tests/clubs/test_admin.py

```diff
@@ -0,0 +1,34 @@
+import json
+import datetime
+
+from django.urls import reverse
+from django.test import TestCase, Client
+
+from users.models import Person
+from clubs.models import Club, Tag
+
+
+class AdminTestCase(TestCase):
+    def setUp(self):
+        self.client = Client()
+
+        self.user1 = Person.objects.create_user('jadams', 'jadams@sas.upenn.edu', 'test')
+        self.user1.first_name = 'John'
+        self.user1.last_name = 'Adams'
+        self.user1.is_staff = True
+        self.user1.is_superuser = True
+        self.user1.save()
+
+        self.tag1 = Tag.objects.create(name='Engineering')
+        self.club1 = Club.objects.create(id='penn-labs', name='Penn Labs')
+        self.club1.tags.add(self.tag1)
+
+    def test_admin_views(self):
+        """
+        Ensure that all admin pages are able to render correctly.
+        """
+        self.client.login(username=self.user1.username, password='test')
+
+        for page in ['admin:index', 'admin:clubs_club_changelist', 'admin:clubs_favorite_changelist', 'admin:clubs_tag_changelist', 'admin:clubs_membership_changelist']:
+            resp = self.client.get(reverse(page))
```

Comment:
Could we also test to see if some text that we expect is on the page? Just cuz a 200 status code does t necessarily mean we’re rendering the right thing.

Reply:
All of these paths are handled by Django, so any bugs with routing would actually be upstream (we don't handle any of this URL routing or view rendering).

The checks just ensure that the `ModelAdmin` interaction fields don't throw a 500 or similar error, and also to keep the test coverage high (alternative would be whitelisting `admin.py`, which I don't think is a great idea).